papers/Collabra/cover_letter.Rmd

```diff
@@ -2,21 +2,24 @@
 output: pdf_document
 ---
 
-Dear Editors,
+Dear Editor,
 
-We are pleased to submit this paper to *Mathematics*, for the special issue "Advances in Statistical Computing".
+We are pleased to submit this paper to [TODO].
 
-The paper, titled "Check your outliers! An accessible introduction to identifying statistical outliers in R with *easystats*", provides an overview of current recommendations and best practices regarding the diagnosis and treament of outliers. It demonstrates how these recommendations can be easily and conveniently implemented in the R software using the *{performance}* package of the *easystats* ecosystem. The manuscript covers univariate, multivariate, and model-based statistical outlier detection methods, their recommended threshold, standard output, and plotting method, among other things.
+The paper, titled "Check your outliers! An accessible introduction to identifying statistical outliers in R with *easystats*", provides an overview of current recommendations and best practices regarding the diagnosis and treatment of outliers, a common issue faced by researchers - and a potential source of scientific malpractice. 
 
-In this sense, the paper fits very well with the special issue "Advances in Statistical Computing", as it essentially communicates to the wider public current advances in the statistical computing of outlier detection algorithms and their implementation in currently available open source and free software. This makes the manuscript relevant to data science, behavioural science, and statistical computing more generally.
+It explains the key approaches and highlights recommendations, and shows how users can adopt them in their R analysis with just one function. The manuscript covers univariate, multivariate, and model-based statistical outlier detection methods, their recommended threshold, standard output, and plotting method, among other things.
+
+Beyond acting like a concise review of outlier treatment procedures and practical tutorial, we also describe a new method (a consensus-based approach) and discuss its benefits ad limitations. In this sense, the paper fits very well with the scope of the journal, as it essentially communicates to the wider public current advances in the statistical computing of outlier detection algorithms and their implementation in currently available open source and free software. This makes the manuscript relevant to data science, behavioural science, and statistical computing more generally. 
```

Comment:
Actually we don't discuss benefits and limitations of the consensus-based approach right now. @DominiqueMakowski actually are the method's benefits and limitations?